tests/unit/amo/sagas/test_users.js

```diff
@@ -37,78 +39,148 @@ describe(__filename, () => {
+        });
+    });
   });
 
-  it('calls the API to fetch user after fetchUserAccount()', async () => {
-    const user = createUserAccountResponse();
+  describe('loadUserAccount', () => {
+    it('calls the API to fetch user after fetchUserAccount()', async () => {
+      const user = createUserAccountResponse();
+
+      mockApi
+        .expects('userAccount')
+        .once()
+        .returns(Promise.resolve(user));
 
-    mockApi
-      .expects('userAccount')
-      .once()
-      .returns(Promise.resolve(user));
+      sagaTester.dispatch(fetchUserAccount({
+        errorHandlerId: errorHandler.id,
+        username: 'tofumatt',
+      }));
 
-    sagaTester.dispatch(fetchUserAccount({
-      errorHandlerId: errorHandler.id,
-      username: 'tofumatt',
-    }));
+      const expectedCalledAction = loadUserAccount({ user });
 
-    const expectedCalledAction = loadUserAccount({ user });
+      await sagaTester.waitFor(expectedCalledAction.type);
+      mockApi.verify();
 
-    await sagaTester.waitFor(expectedCalledAction.type);
-    mockApi.verify();
+      const calledAction = sagaTester.getCalledActions()[2];
+      expect(calledAction).toEqual(expectedCalledAction);
+    });
+
+    it('dispatches an error', async () => {
+      const error = new Error('a bad API error');
+      mockApi
+        .expects('userAccount')
+        .returns(Promise.reject(error));
+
+      sagaTester.dispatch(fetchUserAccount({
+        errorHandlerId: errorHandler.id,
+        username: 'tofumatt',
+      }));
 
-    const calledAction = sagaTester.getCalledActions()[2];
-    expect(calledAction).toEqual(expectedCalledAction);
+      const errorAction = errorHandler.createErrorAction(error);
+      await sagaTester.waitFor(errorAction.type);
+      expect(sagaTester.getCalledActions()[2]).toEqual(errorAction);
+    });
   });
 
-  it('dispatches an error', async () => {
-    const error = new Error('a bad API error');
-    mockApi
-      .expects('userAccount')
-      .returns(Promise.reject(error));
+  describe('editUserAccount', () => {
+    it('calls the API to edit a user after editUserAccount()', async () => {
+      const user = createUserAccountResponse({ id: 5001 });
+      const userFields = {
+        biography: 'I fell into a burning ring of fire.',
+        location: 'Folsom Prison',
+      };
+
+      mockApi
+        .expects('editUserAccount')
+        .once()
+        .returns(Promise.resolve(user));
+
+      sagaTester.dispatch(editUserAccount({
+        errorHandlerId: errorHandler.id,
+        userFields,
+        userId: user.id,
+      }));
+
+      const expectedCalledAction = editUserAccount({
+        errorHandlerId: errorHandler.id,
+        userFields,
+        userId: user.id,
+      });
+
+      const calledAction = await sagaTester.waitFor(expectedCalledAction.type);
+
+      mockApi.verify();
+      expect(calledAction).toEqual(expectedCalledAction);
+
+      // Make sure the finish action is also called.
```

Comment:
We should also verify that `errorHandler.createClearingAction` was called, as that's something else that the saga code does.